addons/rose/addon/components/rose/form/radio/card/wide/index.js

```diff
@@ -0,0 +1,7 @@
+import RoseFormRadioRadioComponent from '../../radio';
+
+export default class RoseFormRadioRadioCardComponent extends RoseFormRadioRadioComponent {
```

Comment:
I see this more as a different style of the card component, rather than an entirely different component.  Open to discussing.

Reply:
good point! I think that helps contain the layouts of this component.  we can pass in a component attribute such as `layout` and this can be easily extended to other layouts if we decide to add more card styles.